cmd/kubelet/app/server.go

```diff
@@ -400,6 +407,33 @@ func UnsecuredDependencies(s *options.KubeletServer, featureGate featuregate.Fea
 	if err != nil {
 		return nil, err
 	}
+	var tp oteltrace.TracerProvider
+	if utilfeature.DefaultFeatureGate.Enabled(features.KubeletTracing) {
+	    hostname, err := nodeutil.GetHostname(s.HostnameOverride)
+	    if err != nil {
+		    return nil, err
+	    }
+	    opts := []otlpgrpc.Option{}
+		tc := componentbaseconfigv1alpha1.TracingConfiguration{
```

Comment:
isn't `tc` the same as `s.KubeletConfiguration.Tracing`?

Reply:
yes, i must've been tired